components/App.js

```diff
@@ -0,0 +1,87 @@
+import TodoList from './TodoList';
+import TodoInput from './TodoInput';
+import TodoCount from './TodoCount';
+import TodoFilter from './TodoFilter';
+import { TODOS, ALL, ACTIVE, COMPLETED } from '../constants';
+
+class App {
+  constructor() {
+    this.todos = [];
+    this.todosVisible = [];
+    this.whatToShow = ALL;
```

Comment:
```suggestion
class App {

  todos; todosVisible; whatToShow;
  
  constructor() {
    this.todos = [];
    this.todosVisible = [];
    this.whatToShow = ALL;
```
이런식으로 멤버 변수를 미리 선언해주면 나중에 이 클래스가 어떤 프로퍼티를 사용하고 있는지 명시할 수 있어서 좋습니다!

Reply:
확실히 명시적으로 선언하는 것이 가독성에 좋은 것 같습니다!